app/database/__init__.py

```diff
@@ -34,7 +38,7 @@ class Role(SQLModel, table=True):
 class Server(SQLModel, table=True):
     snowflake: int = Field(unique=True, primary_key=True, nullable=False)
     channels: Optional[str] = "[]"
-    users: str = Field(nullable=False)
+    users: List["DBUser"] = Relationship(back_populates="users") #Field(nullable=False)
```

Comment:
This doesn't work, you'd have to use a JSON type and not a relationship

Reply:
or pickle it and store it as a blob or stringify it and store it as a.... string